Cargo.toml

```diff
@@ -518,7 +518,7 @@ single_range_in_vec_init = "allow"
 
 # There are a bunch of rules currently failing in the `style` group, so
 # allow all of those, for now.
-style = "allow"
+style = { level = "allow", priority = -1 }
```

Comment:
For my understanding, what does `priority = -1` do here?

Reply:
TL;DR:
```
error: lint group `style` has the same priority (0) as a lint
   --> Cargo.toml:521:1
    |
521 | style = { level = "allow"}
    | ^^^^^   ------------------ has an implicit priority of 0
...
524 | almost_complete_range = "allow"
    | --------------------- has the same priority as this lint
    |
    = note: the order of the lints in the table is ignored by Cargo
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#lint_groups_priority
    = note: `#[deny(clippy::lint_groups_priority)]` on by default
help: to have lints override the group set `style` to a lower priority
    |
521 | style = { level = "allow", priority = -1 }
```

So in short, it seems that it explicitly gives `style` group lower priority, in case other items in the lints table wanted to override it (say we wanted to opt into a single lint from the `style` group). It's a bit silly, because we do not opt into style lints, but that's probably on the lint implementation and not on us.

Reply:
It looks like in the lint implementation, they simply look for a group specifier (`style`) and any other specifier (`almost_complete_range`) that has the same priority.
https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/cargo/lint_groups_priority.rs

Reply:
Thanks for the explanation! That's good to know.